### PR TITLE
feat(Accordion): added toggle alignment functionality

### LINKED ISSUE #9786: Accordion - support for toggle on the left/start
Core PR - https://github.com/patternfly/patternfly/pull/5987

* Adds `.pf-m-toggle-start` to the parent `.pf-v5-c-accordion` element to indicate the toggle should be on the left/start.
* The toggle icon should come before the toggle text in the HTML when this variant is set.

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #9786

[Accordion toggle icon at start example preview](https://patternfly-react-pr-9877.surge.sh/components/accordion#toggle-icon-at-start)

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -18,6 +18,8 @@ export interface AccordionProps extends React.HTMLProps<HTMLDListElement> {
   isBordered?: boolean;
   /** Display size variant. */
   displaySize?: 'default' | 'lg';
+  /** Alignment of the toggle icon. */
+  toggleAlignment?: 'start' | 'end';
```
Comment (mcoker) [inline packages/react-core/src/components/Accordion/Accordion.tsx:22]: Just a comment, but I might refer to this as "position"? The button uses `iconPosition` for start/end. Not sure if there is a standard for naming this kind of prop.

```suggestion
  /** Position of the toggle icon. */
  togglePosition?: 'start' | 'end';
```

**Review by mcoker — APPROVED** [review]
LGTM! Just one question about the pop name, but I'd say go with whatever is consistent there. One argument for using "position" is that it is moving something to come before or after, where "alignment" to me anyways just aligns a thing to the left/right/top/bottom/center/baseline/etc in the available space where it already is.

File: packages/react-core/src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (thatblindgeye) [reply packages/react-core/src/components/Accordion/Accordion.tsx:22]: That sounds much better to me! Will push an update

**Review by nicolethoen — APPROVED** [review]

**Review by andrew-ronaldson — APPROVED** [review]
LGTM, I can create some design kit issues for v5/6

**Review by mfrances17 — APPROVED** [review]
👍

**Pull request merged**